app/theme/client/imports/components/sidebar/sidebar.css

```diff
@@ -56,6 +56,7 @@
 		& img {
 			max-width: 100%;
 			height: 100%;
+			margin-left : 17%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace before ":" (declaration-colon-space-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5130324)

Reply:
ok i will fix it